packages/domain-picker/src/hooks/use-domain-suggestions.ts

```diff
@@ -34,8 +37,11 @@ export function useDomainSuggestions(
 			if ( ! domainSearch || domainSearch.length < DOMAIN_QUERY_MINIMUM_LENGTH ) {
 				return;
 			}
-			const { getDomainSuggestions, getDomainState, getDomainErrorMessage } =
-				select( DOMAIN_SUGGESTIONS_STORE );
+			const {
+				getDomainSuggestions,
+				getDomainState,
+				getDomainErrorMessage,
+			}: DomainSuggestionsSelect = select( DOMAIN_SUGGESTIONS_STORE );
```

Comment:
I wonder if `locale` should be listed as a dependency since it's one of the variable arguments in `allDomainSuggestions`.